src/rules/color-hex-length/README.md

```diff
@@ -0,0 +1,59 @@
+# color-hex-length
+
+Specify short or long notation for hex colors.
+
+```css
+    a { color: #fff }
+/**              ↑
+ * These hex colors */
+```
+
+## Options
+
+`string`: `"short"|"long"`
```

Comment:
Instead of these unique keywords, how would you feel about renaming and using always/never?

We could do something like `color-hex-short` with always/never option?